src/test/java/software/amazon/awssdk/crt/test/Elasticurl.java

```diff
@@ -0,0 +1,362 @@
+        exit();
+    }
+
+    static CommandLine parseArgs(String args[]) {
+        Options cliOpts = new Options();
+
+        cliOpts.addOption("h", "help", false, "show this help message and exit");
+        cliOpts.addOption(Option.builder().longOpt("cacert").hasArg().argName("file")
+                .desc("path to a CA certificate file.").build());
+        cliOpts.addOption(Option.builder().longOpt("capath").hasArg().argName("dir")
+                .desc("path to a directory containing CA files.").build());
+        cliOpts.addOption(Option.builder().longOpt("cert").hasArg().argName("file")
+                .desc("path to a PEM encoded certificate to use with mTLS.").build());
+        cliOpts.addOption(Option.builder().longOpt("key").hasArg().argName("file")
+                .desc("path to a PEM encoded private key that matches cert.").build());
+        cliOpts.addOption(Option.builder().longOpt("connect_timeout").hasArg().argName("int")
+                .desc("time in milliseconds to wait for a connection.").build());
+        cliOpts.addOption(Option.builder("H").longOpt("header").hasArgs().argName("str")
+                .desc("line to send as a header in format 'name:value'. May be specified multiple times.").build());
+        cliOpts.addOption(Option.builder("d").longOpt("data").hasArg().argName("str")
+                .desc("data to send in POST or PUT.").build());
+        cliOpts.addOption(Option.builder().longOpt("data_file").hasArg().argName("file")
+                .desc("file to send in POST or PUT.").build());
+        cliOpts.addOption(Option.builder("M").longOpt("method").hasArg().argName("str")
+                .desc("request method. Default is GET)").build());
+        cliOpts.addOption("G", "get", false, "uses GET for request method.");
+        cliOpts.addOption("P", "post", false, "uses POST for request method.");
+        cliOpts.addOption("I", "head", false, "uses HEAD for request method.");
+        cliOpts.addOption("i", "include", false, "includes headers in output.");
+        cliOpts.addOption("k", "insecure", false, "turns off x.509 validation.");
+        cliOpts.addOption(Option.builder("o").longOpt("output").hasArg().argName("file")
+                .desc("dumps content-body to FILE instead of stdout.").build());
+        cliOpts.addOption(Option.builder("t").longOpt("trace").hasArg().argName("file")
+                .desc("dumps logs to FILE instead of stderr.").build());
+        cliOpts.addOption(Option.builder("p").longOpt("alpn").hasArgs().argName("str")
+                .desc("protocol for ALPN. May be specified multiple times.").build());
+        cliOpts.addOption(null, "http1_1", false, "HTTP/1.1 connection required.");
+        cliOpts.addOption(null, "http2", false, "HTTP/2 connection required.");
+        cliOpts.addOption(Option.builder("v").longOpt("verbose").hasArg().argName("str")
+                .desc("logging level (ERROR|INFO|DEBUG|TRACE) default is none.").build());
```

Comment:
trivial: there's "WARN" too. not sure if the other Elasticurls support that

Reply:
Nope, all the others don't have `WARN`...